message_center_compassion/tools/onramp_connector.py

```diff
@@ -42,14 +42,14 @@ def __new__(cls):
             connect_url = config.get('connect_url')
             api_key = config.get('connect_api_key')
             if connect_url and api_key:
-                OnrampConnector.__instance._connect_url = connect_url
-                OnrampConnector.__instance._api_key = api_key
+                OnrampConnector._connect_url = connect_url
+                OnrampConnector._api_key = api_key
```

Comment:
Are you sure about this? Is it a change from python3? I want to make sure we access the correct instance of the class (this is a Singleton pattern).

Reply:
I went through the debugger and it always told me that OnrampConnector had no __instance parameter. The program crashed if I didn't do that. But it may be due to something else

Reply:
So... I put it back and now everything seems to be working.......